src/wmnds/patterns/cookies/_cookies.scss

```diff
@@ -18,3 +18,7 @@
     }
   }
 }
+
+.wmnds-website-code-example__njk .wmnds-cookies-banner {
+  display: block;
+}
```

Comment:
This should be moved into the website's scss. As the end user won't benefit from this CSS being shipped with the design system.